servatrice/src/servatrice_database_interface.cpp

```diff
@@ -812,8 +812,10 @@ void Servatrice_DatabaseInterface::logMessage(const int senderId, const QString
     execSqlQuery(query);
 }
 
-bool Servatrice_DatabaseInterface::changeUserPassword(const QString &user, const QString &oldPassword, const QString &newPassword)
+bool Servatrice_DatabaseInterface::changeUserPassword(const QString &user, const QString &oldPassword, const QString &newPassword, const bool &validateOldPassword)
 {
+	//note: function returns false if password is successfully changed
```

Comment:
Yep, my fault in first trying to implement a POSIX-like return value (0 = no error, > 0 = errno), then not using it  and leaving such code there.